src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const cssProperties = sourceString
+    .trim()
+    .split(';')
+    .filter((str) => str.trim())
+    .map((str) => {
+      return str
+        .trim()
+        .split(':')
+        .map((s) => s.trim());
```

Comment:
The chained methods should have the same indentation to improve readability. Align the chained methods `.trim()`, `.split(';')`, `.filter(...)`, and `.map(...)` vertically.